questionpy_sdk/commands/i18n/_init.py

```diff
@@ -0,0 +1,110 @@
+from collections.abc import Collection
+from pathlib import Path
+
+import babel.messages.extract
+import babel.messages.frontend
+import babel.messages.pofile
+import click
+from click import ClickException
+
+from questionpy.i18n import domain_of
+from questionpy_common.manifest import Bcp47LanguageTag
+from questionpy_sdk._i18n_utils import bcp47_to_posix
+from questionpy_sdk.commands.i18n import extract
+from questionpy_sdk.package.source import PackageSource
+
+
+def _init_explicit(ctx: click.Context, pot: Path, localedir: Path, locale: Bcp47LanguageTag, *, force: bool) -> None:
+    init_cmd = babel.messages.frontend.InitCatalog()
+    init_cmd.locale = bcp47_to_posix(locale)
+    init_cmd.input_file = pot
+    init_cmd.output_file = localedir / f"{locale}.po"
+
+    if init_cmd.output_file.exists() and not force:
+        update_cmd = f"{ctx.parent.command_path} update" if ctx.parent else "update"
+        msg = (
+            f"Output file '{init_cmd.output_file}' already exists. Use '{update_cmd}' to update existing translations "
+            f"or pass '--force' to overwrite."
+        )
+        raise click.ClickException(msg)
+
+    init_cmd.ensure_finalized()
+    init_cmd.run()
+
+
+def _init_in_source_dir(ctx: click.Context, package: PackageSource, locale: Bcp47LanguageTag, *, force: bool) -> None:
+    domain = domain_of(package.config)
+
+    pot_file = package.path / "locale" / f"{domain}.pot"
+    if not pot_file.exists():
+        ctx.invoke(extract, source=package.path)
+
+    output_mo_file = package.path / "locale" / f"{locale}.po"
+    if output_mo_file.exists() and not force:
+        update_cmd = f"{ctx.parent.command_path} update" if ctx.parent else "update"
+        msg = (
+            f"Output file '{output_mo_file}' already exists. Use '{update_cmd}' to update existing translations "
+            f"or pass '--force' to overwrite."
+        )
+        raise click.ClickException(msg)
```

Comment:
Passiert dieser Check nicht auch nochmal in `_init_explicit`, wodurch er dann hier entfernt werden könnte?

Reply:
Ah ja. Dabei fiel mir auf, dass `init` fast alles macht, daher habe ich einiges in `_init_in_source_dir` verschoben: 2a8c953352626f9335fb29dd11aaca7c2e92d71a